packages/cli/src/services/util.ts

```diff
@@ -37,8 +41,19 @@ export async function walkDirectory (
 
 export async function loadJsFile (filepath: string): Promise<any> {
   try {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
-    let exported = require(filepath)
+    let exported = await require(filepath)
+    if (exported instanceof Function) {
+      exported = await exported()
+    }
+    return exported
+  } catch (err: any) {
+    throw new Error(`Error loading file ${filepath}\n${err.stack}`)
+  }
+}
+
+export async function loadMjsFile (filepath: string): Promise<any> {
```

Comment:
@nahuelon we need to have a single import for both files. The reason is that if a user has type: module in their package.json, the files don't need to have the extenstion `.mjs`

Reply:
@umutuzgur , done.

Reply:
Should we delete this function now? Since `loadJsFile` now supports

Reply:
@nahuelon ☝️

Reply:
Your are right. Thanks. Done.